### PR TITLE
DOCS-11176 Shopify 1.0

### PR DESCRIPTION
**Pull request by sathya0**

### CODE REVIEW

File: shopify/1.0/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+* Credentials to access the Shopify target resource
+* Anypoint Platform
+* Anypoint Studio version 7.3.5 or later
+
+== Common Use Cases For the Connector
+
+Some common use cases for Shopify Connector include:
+
+* Syncing a new customer's information when they place an order.
```
Comment (dhirsch1) [inline shopify/1.0/modules/ROOT/pages/index.adoc]: Remove the periods (a nit)

File: shopify/1.0/modules/ROOT/pages/shopify-connector-reference.adoc [hunk 2]
```diff
@@ -0,0 +1,3665 @@
...
+| Max Redelivery Count a| Number | Maximum number of times that a redelivered request can be processed unsuccessfully before returning a REDELIVERY_EXHAUSTED error. |  |
+| Message Digest Algorithm a| String | Secure hashing algorithm to use if the *Use Secure Hash* field is `true`. If the payload of the message is a Java object, Mule ignores this value and returns the value that the payload’s `hashCode()` returned. |  |
+| Message Identifier a| <<RedeliveryPolicyMessageIdentifier>> | Defines which strategy is used to identify the messages. |  |
+| Object Store a| ObjectStore | Configures the object store that stores the redelivery counter for each message. |  |
+|===
+
+[[RedeliveryPolicyMessageIdentifier]]
+=== Redelivery Policy Message Identifier
+
+Configures how to identify a relivered message and how to find out when the message was redelivered.
```
Comment (dhirsch1) [inline shopify/1.0/modules/ROOT/pages/shopify-connector-reference.adoc]:
```suggestion
Configures how to identify a redelivered message and how to find out when the message was redelivered.
```

File: shopify/1.0/modules/ROOT/pages/shopify-connector-reference.adoc [hunk 3]
```diff
@@ -0,0 +1,3665 @@
...
+
+** HTTP
+** HTTPS |  Protocol to use for communication. |  HTTP |
+| TLS Configuration a| <<Tls>> | Configures TLS. If using the HTTPS protocol, you must configure TLS.  |  |
+| Reconnection a| <<Reconnection>> |  Configures a reconnection strategy to use when a connector operation fails to connect to an external server. |  |
+| Consumer Key a| String |  OAuth consumer key, as registered with the service provider. |  | x
+| Consumer Secret a| String |  OAuth consumer secret, as registered with the service provider. |  | x
+| Authorization Url a| String |  URL of the service provider's authorization endpoint. |  https://{shop}.myshopify.com/admin/oauth/authorize |
+| Access Token Url a| String |  URL of the service provider's access token endpoint. |  https://{shop}.myshopify.com/admin/oauth/access_token |
+| Scopes a| String |  OAuth scopes to request during the OAuth dance. This value defaults to the scopes in the annotation.  |  read_products write_products read_customers write_customers read_orders write_orders read_all_orders |
```
Comment (dhirsch1) [inline shopify/1.0/modules/ROOT/pages/shopify-connector-reference.adoc]: Do you know what the "annotations" means? There are default scopes in the Default column, so nothing else needs to be said about defaults. May want to check this with a developer.

File: shopify/1.0/modules/ROOT/pages/shopify-connector-reference.adoc [hunk 4]
```diff
@@ -0,0 +1,3665 @@
...
+
+=== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use. | | x
+| ids a| String |  Restrict results to customers specified by a comma-separated list of IDs. |  |
+| Since ID a| Number |  Restrict results to those after the specified ID. |  |
+| Created at min a| String |  Show customers created after a specified date. The format is `2014-04-25T16:15:47-04:00`. |  |
```
Comment (dhirsch1) [inline shopify/1.0/modules/ROOT/pages/shopify-connector-reference.adoc]: Show customers created >  Show the customers created...

Here and in similar descriptions

File: shopify/1.0/modules/ROOT/pages/shopify-connector-reference.adoc [hunk 5]
```diff
@@ -0,0 +1,3665 @@
...
+=== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use. | | x
+| ids a| String |  Retrieve only orders specified by a comma-separated list of order IDs. |  |
+| limit a| Number | Amount of results to return. The maximum is 250. | 50 |
+| Since ID a| Number |  Restrict results to those after the specified ID. |  |
+| Created at min a| String |  Show orders created at or after date. The format is `2014-04-25T16:15:47-04:00`. |  |
```
Comment (dhirsch1) [inline shopify/1.0/modules/ROOT/pages/shopify-connector-reference.adoc]: "...the date specified in this field"

Same for similar descriptions

File: shopify/1.0/modules/ROOT/pages/shopify-connector-reference.adoc [hunk 6]
```diff
@@ -0,0 +1,3665 @@
...
+| Created at max a| String |  Show orders created at or before date. The format is `2014-04-25T16:15:47-04:00`. |  |
+| Updated at min a| String |  Show orders last updated at or after date. The format is `2014-04-25T16:15:47-04:00`. |  |
+| Updated at max a| String |  Show orders last updated at or before date. The format is `2014-04-25T16:15:47-04:00`. |  |
+| Processed at min a| String |  Show orders imported at or after date. The format is `2014-04-25T16:15:47-04:00`. |  |
+| Processed at max a| String |  Show orders imported at or before date. The format is `2014-04-25T16:15:47-04:00`. |  |
+| Attribution app ID a| String |  Show orders attributed to a certain app, specified by the app ID. Set as current to show orders for the app currently consuming the API. |  |
```
Comment (dhirsch1) [inline shopify/1.0/modules/ROOT/pages/shopify-connector-reference.adoc]: The app for which to show orders, as specified by the app ID. 

Should "current" be in back ticks? Is that the actual value to set?

File: shopify/1.0/modules/ROOT/pages/shopify-connector-reference.adoc [hunk 7]
```diff
@@ -0,0 +1,3665 @@
...
+Retrieves a count of products. This operation makes an HTTP GET request to the /admin/api/2021-07/products/count.json endpoint.
+
+
+=== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use. | | x
+| vendor a| String |  Return product count by product title. |  |
```
Comment (dhirsch1) [inline shopify/1.0/modules/ROOT/pages/shopify-connector-reference.adoc]:
```suggestion
| vendor a| String |  Return the product count by product title. |  |
```

Comment (dhirsch1) [reply shopify/1.0/modules/ROOT/pages/shopify-connector-reference.adoc]: Same comment for similiar descriptions

File: shopify/1.0/modules/ROOT/pages/shopify-connector-reference.adoc (hunk 3 above)
Comment (sathya0) [reply shopify/1.0/modules/ROOT/pages/shopify-connector-reference.adoc]: This Scopes parameter is a definition I got from the spreadsheet. It looks like the dev left a comment on the spreadsheet saying the same thing-- that we don't need to say the defaults, so I'll take it out of the definition.

File: shopify/1.0/modules/ROOT/pages/shopify-connector-reference.adoc (hunk 6 above)
Comment (sathya0) [reply shopify/1.0/modules/ROOT/pages/shopify-connector-reference.adoc]: I believe they mean current as in the current attribution app ID, but let me verify with dev

**Review by dhirsch1 — APPROVED** [review]

**Review by dhirsch1 — APPROVED** [review]

**Pull request merged**